checkstyle.xml

```diff
@@ -82,6 +82,11 @@
         <module name="ArrayTypeStyle"/>
         <module name="MissingSwitchDefault"/>
         <module name="FallThrough"/>
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+        <module name="StringLiteralEquality"/>
+        <module name="UnnecessaryParentheses"/>
+        <module name="ExplicitInitializationCheck"/>
```

Comment:
If we're not initializing an object value in the constructor, then that deserves to be called out and made explicit in the code using ` = null`. 

Booleans should *always* be explicitly initialized—the idea that one boolean type is "default" is an anachronistic holdover from C and not at all obvious to most people.